.travis.yml

```diff
@@ -31,7 +31,7 @@ script:
 
 deploy:
   provider: script
-  skip_cleanup: true
+  skip_cleanup: false
```

Comment:
Why the difference here?

Reply:
Preparation for the release. Not crucial since we use vendor, but we want the system to do a "git stash" to remove all possible local changes when running the script because otherwise goreleaser would fail to build the executables since it first ensures that git status is clean.
I can put this in the next PR for the release if you want.

Reply:
imo fine in this PR.  Thanks!